internal/webserver/webserver.go

```diff
@@ -230,19 +231,20 @@ func (n kubeFilter) isNodeListing(fn http.HandlerFunc) http.HandlerFunc {
 }
 
 func (n kubeFilter) Start(ctx context.Context) error {
-	http.HandleFunc("/api/v1/namespaces", func(writer http.ResponseWriter, request *http.Request) {
+	r := mux.NewRouter()
+	r.PathPrefix("/api/v1/namespaces").HandlerFunc(func(writer http.ResponseWriter, request *http.Request) {
```

Comment:
I was thinking about the need for `PathPrefix` here, as I suggested to you, but I guess we don't need it: we are solving the missing listing capabilities at the cluster lever for the Tenant Owner, and that's good, but we don't have to decorate the request for a specific resource (`/api/v1/namespaces/{namespace}`) since the Tenant Owner will be granted by the `capsule` operator using the proper RBAC.

I think we can safely switch to `r.HandleFunc`.

Reply:
Ok :+1: